lib/routes/hottoys/index.ts

```diff
@@ -0,0 +1,60 @@
+import { Route } from '@/types';
+import { load } from 'cheerio';
+import puppeteer from '@/utils/puppeteer';
+
+export const route: Route = {
```

Comment:
Please set [`features.requirePuppeteer`](https://docs.rsshub.app/joinus/new-rss/start-code#creating-route) to true.

Reply:
fixed